src/mpid/ch4/netmod/ofi/ofi_events.c

```diff
@@ -597,6 +597,7 @@ static int am_recv_event(struct fi_cq_tagged_entry *wc, MPIR_Request * rreq)
      * when that's the case */
 #define MAX_HDR_SIZE 256        /* need accommodate MPIDI_AMTYPE_RDMA_READ */
     char temp[MAX_HDR_SIZE] MPL_ATTR_ALIGNED(MAX_ALIGNMENT);
+    char amtype_rdma_payload[sizeof(MPIDI_OFI_lmt_msg_payload_t)];
     if ((intptr_t) am_hdr & (MAX_ALIGNMENT - 1)) {
```

Comment:
Why don't we just define a variable `MPIDI_OFI_lmt_msg_payload_t  temp_rdma_payload` instead of `char ***`?

Reply:
yeah, sounds good. I will change it.